modules/post/multi/gather/rsyncd_creds.rb

```diff
@@ -0,0 +1,90 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Post
+  include Msf::Post::File
+  include Msf::Post::Unix
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name'          => 'UNIX Gather RSYNC Credentials',
+        'Description'   => %q(
+          Post Module to obtain credentials saved for RSYNC in various locations
+        ),
+        'License'       => MSF_LICENSE,
+        'Author'        => [ 'Jon Hart <jon_hart[at]rapid7.com>' ],
+        'Platform'      => %w(bsd linux osx unix),
```

Comment:
I personally think if the user knows the exact location of the file, he can just download the file through meterpreter download command. Further, as you noted rsync won't be installed on Windows most of the time. Being the author of the module I think it is your call :)